services/keyword/create_keyword_result.go

```diff
@@ -0,0 +1,154 @@
+package keyword
+
+import (
+	"fmt"
+	"net/url"
+
+	. "go-crawler-challenge/forms"
```

Comment:
It's unexpected to import forms here just for the validation error messages. How about instead having constants for services (`/services/constants.go`) or maybe better just inlining them in this service file? To me, the latter seems better as I am not sure if the validation messages can really be shared.

Moving forward, having a locale file like in Rails might be the best option 💭